server/src/items/classes/spell_scrolls/SpellScroll.js

```diff
@@ -0,0 +1,31 @@
+const Item = require("../Item");
+
+class SpellScroll extends Item {
+
+    use (){
+        const { owner } = this;
+        if (this.useEnergyCost && owner.energy < this.useEnergyCost) return;
+        if (this.useGloryCost && owner.glory < this.useGloryCost) return;
+
+        super.use();
+
+        if (this.useEnergyCost) owner.modEnergy(-this.useEnergyCost);
+        if (this.useGloryCost) owner.modGlory(-this.useGloryCost);
+    }
+
+    onUsed() {
+        console.log("SpellScroll used")
+        super.onUsed();
+    }
+    
+    getBoardTilesInRange(inputRange) {
+        return this.owner.board.getTilesInEntityRange(this.owner, 1);
```

Comment:
`inputRange` argument is not used here.

`getTilesInEntityRange(this.owner, inputRange || 1);` ?